### DISCUSSION TITLE
How to specify Android certificate store

### DISCUSSION THREAD

**Discussion by URNOTCharlotte** [microsoft/msquic#4622] [Q&A] [answered]
`QUIC_CREDENTIAL_FLAG_NO_CERTIFICATE_VALIDATION`

Indicate to the TLS layer that no server certificate validation is to be performed. **THIS IS DANGEROUS; DO NOT USE IN PRODUCTION**

If setting the "QUIC_CREDENTIAL_FLAG_NO_CERTIFICATE_VALIDATION" flag for client, server certificate validation will be disable.

I want to enable the server certificate validation for client, but I don't know how to specify certificate store for android device. Is there some interface that does the following?
void SSL_CTX_set_cert_store(SSL_CTX *, X509_STORE *);

**Comment by nibanks** [comment, marked as answer]
We don't (yet) support the Android certificate store. We'd be happy to review any PR you put together.
